cfpb/design-system
# Update frames of reference activity

Incorporate the new frames of reference activity template into the equity-centered design guidelines. See GHE/Design-Development/Design-Thinking-and-User-Research/issues/58 for more details.

## Additions

- PNG and PDF versions of the frames of reference activity template
- A `special-page` page with the Mural version of the frames of reference activity embedded

## Changes

- Updated frames of reference content
- Updated link to 18F's frames of reference presentation that I missed in #1504

## Testing

1. See if the new content looks OK
2. Make sure the links to the PDF, PNG, and Mural versions of the template work

## Screenshots

Updated activity content | New Mural template page
--- | ---
![foundations-page](https://user-images.githubusercontent.com/1862695/206791568-e4a2e691-ac30-4dc2-976a-68c18b87163b.png) | ![mural-page](https://user-images.githubusercontent.com/1862695/206791577-f7b576c4-f5ab-4833-be51-b289af0e7a8a.png)

## Checklist
